chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -237,6 +238,8 @@ public static void main(final String[] args) {
             break;
         }
       } catch (Throwable t) {
+        // set receiver in case an exception was thrown before it was set in one of the start modes.
+        Log.setReceiver(ConsoleReceiver.INSTANCE, Level.INFO, Level.WARNING, Level.ERROR);
```

Comment:
This would always override the receiver. Can we change it so that it only does if no other receiver is registered? Or maybe just set this receiver as default right at the top of main?

Reply:
The default receiver is the buffered one which doesn't log anything and buffers the messages until a non-buffering receiver is set. So early messages are logged in the gui and not silently sent to the console.

The only case this `catch` block is if something fails in initialising jfx/headless in which case the respective receiver may not be set or even valid in the case of jfx